src/lib/voiceSearchHelper/index.ts

```diff
@@ -139,22 +109,14 @@ export default function createVoiceSearchHelper({
     resetState('finished');
   };
 
-  const toggleListening = (): void => {
-    if (!isBrowserSupported()) {
-      return;
-    }
-    if (isListening()) {
-      stop();
-    } else {
-      start();
-    }
-  };
-
   return {
     getState,
     isBrowserSupported,
     isListening,
-    toggleListening,
+    startListening: start,
```

Comment:
We can rename our internal functions `startListening` and `stopListening` to use the same vocabulary. Wdyt?

Reply:
https://github.com/algolia/instantsearch.js/pull/4363/commits/9fe79ab80fd29595839a5364cd9939821ffe9d2e